examples/log_slider.py

```diff
@@ -0,0 +1,14 @@
+"""Simple demonstration of magicgui."""
```

Comment:
If I start moving the slider I get the following error

```python-traceback
(base) czirwc1macos2701:magicgui nsofroniew$ python examples/log_slider.py
Traceback (most recent call last):
  File "/Users/nsofroniew/GitHub/magicgui/magicgui/widgets/_bases.py", line 336, in <lambda>
    lambda *x: self.changed(value=x[0] if x else None)
  File "/Users/nsofroniew/GitHub/magicgui/magicgui/events.py", line 603, in __call__
    self._invoke_callback(cb, event)
  File "/Users/nsofroniew/GitHub/magicgui/magicgui/events.py", line 625, in _invoke_callback
    cb_event=(cb, event),
  File "/Users/nsofroniew/GitHub/magicgui/magicgui/events.py", line 619, in _invoke_callback
    cb(event)
  File "/Users/nsofroniew/GitHub/magicgui/magicgui/widgets/_bases.py", line 812, in <lambda>
    widget.changed.connect(lambda x: self.changed(value=self))
  File "/Users/nsofroniew/GitHub/magicgui/magicgui/events.py", line 576, in __call__
    raise RuntimeError("EventEmitter loop detected!")
RuntimeError: EventEmitter loop detected!
Abort trap: 6
```

Reply:
hmm. this is fixed by commenting out the `if self._emitting: raise RuntimeError` in events.py... and nothing else seems to break, but I need to investigate a bit more

Reply:
fixed (this was caused by the result widget emitting a signal as the `auto_call` functionality was updating it)